boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -74,6 +74,25 @@ NtLdrOutputLoadMsg(
     }
 }
 
+PACPISDTHeader
+FindAcpiTable()
```

Comment:
Is it needed to kind of duplicate this code, or could we follow-up on what `DetectAcpiBios()` already did?